src/message.ts

```diff
@@ -613,6 +625,40 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<any>
+  public forward(contact: Contact): Promise<any>
+  public forward(id: string): Promise<any>
+  public forward(sendTo: string|Room|Contact): Promise<any> {
+    const m = <MsgRawObj>this.rawObj
```

Comment:
That is because `this.rawObj` might be `undefined`.

So you need to check `undefined` instead of force it to be `MsgRawObj`.

Add the follow code at the begining of the function, then you will be able to use `this.rawObj` because that will make sure it is not `undefined`.

```ts
if (!this.rawObj) {
  throw new Error('no rawObj!')
}
```